internal/criocli/criocli.go

```diff
@@ -130,15 +130,42 @@ func mergeConfig(config *libconfig.Config, ctx *cli.Context) error {
 	if ctx.IsSet("decryption-keys-path") {
 		config.DecryptionKeysPath = ctx.String("decryption-keys-path")
 	}
-
 	if ctx.IsSet("runtimes") {
+		type storageOpts struct {
+			storage string
+			opts    []string
+		}
 		runtimes := StringSliceTrySplit(ctx, "runtimes")
+		storageRuntimeMap := make(map[string]storageOpts)
+		if ctx.IsSet("runtime-storage") {
+			for _, s := range StringSliceTrySplit(ctx, "runtime-storage") {
+				fields := strings.Split(s, ":")
+				if len(fields) < 2 {
+					return fmt.Errorf("invalid format runtime-storage: %s", s)
+				}
+				var opts []string
+				if len(fields) > 3 {
```

Comment:
should this be?
```suggestion
				if len(fields) >= 3 {
```
Your testing prior for `< 2`

Reply:
You are perfectly right, thanks!